sdk/monitor/azure-monitor-ingestion/src/main/java/com/azure/monitor/ingestion/package-info.java

```diff
@@ -2,56 +2,118 @@
+ *     <a href="https://learn.microsoft.com/azure/azure-monitor/essentials/data-collection-endpoint-overview?tabs=portal">This article</a>
+ *     provides an overview of data collection endpoints including their contents and structure and how you can create and work with them.
+ * </p>
  *
  * <p><strong>Data Collection Rule</strong></p>
+ *
  * <p>
- * Data collection rules (DCR) define data collected by Azure Monitor and specify how and where that data should be sent
- * or stored. The REST API call must specify a DCR to use. A single DCE can support multiple DCRs, so you can specify a different DCR for different sources and target tables.
+ *     Data collection rules (DCR) define data collected by Azure Monitor and specify how and where that data should be sent
+ *     or stored. The REST API call must specify a DCR to use. A single DCE can support multiple DCRs, so you can specify a different DCR for different sources and target tables.
+ * </p>
+ *
  * <p>
- * The DCR must understand the structure of the input data and the structure of the target table. If the two don't match,
- * it can use a transformation to convert the source data to match the target table. You may also use the transform to filter source data and perform any other calculations or conversions.
+ *     The DCR must understand the structure of the input data and the structure of the target table. If the two don't match,
+ *     it can use a transformation to convert the source data to match the target table. You may also use the transform
+ *     to filter source data and perform any other calculations or conversions.
+ * </p>
+ *
  * <p>
- * For more details, see <a href="https://learn.microsoft.com/azure/azure-monitor/essentials/data-collection-rule-overview">Data collection rules</a>
- * in Azure Monitor. For information on how to retrieve a DCR ID,
- * <a href="https://learn.microsoft.com/azure/azure-monitor/logs/tutorial-logs-ingestion-portal#collect-information-from-the-dcr">see this tutorial</a>.
+ *     For more details, see <a href="https://learn.microsoft.com/azure/azure-monitor/essentials/data-collection-rule-overview">Data collection rules</a>
+ *     in Azure Monitor. For information on how to retrieve a DCR ID,
+ *     <a href="https://learn.microsoft.com/azure/azure-monitor/logs/tutorial-logs-ingestion-portal#collect-information-from-the-dcr">see this tutorial</a>.
+ * </p>
  *
  * <p><strong>Log Analytics Workspace Tables</strong></p>
- * Custom logs can send data to any custom table that you create and to certain built-in tables in your Log Analytics
- * workspace. The target table must exist before you can send data to it. The following built-in tables are currently supported:
+ *
+ * <p>
+ *     Custom logs can send data to any custom table that you create and to certain built-in tables in your Log Analytics
+ *     workspace. The target table must exist before you can send data to it. The following built-in tables are currently supported:
```

Comment:
Instead of listing the built-in tables here, link to https://learn.microsoft.com/azure/azure-monitor/logs/logs-ingestion-api-overview#supported-tables. The list is likely to change over time, and the doc I linked to should be the source of truth.